css/index.css

```diff
@@ -104,14 +142,50 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+.teal {
```

Comment:
good job adding classes to just change the color rather than copy/pasting the entire box class over and over agin